server/verify.go

```diff
@@ -372,3 +376,48 @@ func VerifyGceTechnology(attestation *pb.Attestation, tech pb.GCEConfidentialTec
 	}
 	return fmt.Errorf("unknown GCEConfidentialTechnology: %v", tech)
 }
+
+// parseTCGEventLog is a wrapper function around `parsePCClientEventLog` method to:
+// 1. parse partial machine state from TPM TCG event logs.
+// 2. verify GceTechnology since the GCE Technology event is directly related to the TPM.
+func parseTCGEventLog(attestation *pb.Attestation, pcrs *tpmpb.PCRs, opts VerifyOpts) (*pb.MachineState, error) {
```

Comment:
I liked your suggestion of `parseMachineStateFromTPM` better than `parseTCGEventLog`. Since it is also doing event log replay and event parsing. The term "parse" is a bit overloaded unfortunately.

Reply:
Done.